docs/first-order/classify_waveforms.ipynb

```diff
@@ -0,0 +1,2327 @@
+{
```

Comment:
<p>mention it in the markdown that they can change censor_period, stim_duration, time_resolution, and window times. Also would be best to put those in their own cell above this one. Probably should also add a new section heading to explain this stuff? in the heading you can also briefly explain what this code is doing, and mention the purpose of the function 'show_counts' and what its output means</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>